tests/MongoDB.Driver.Examples/TransactionExamplesForDocs/WithTransactionExample1.cs

```diff
@@ -0,0 +1,100 @@
+﻿/* Copyright 2019-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using FluentAssertions;
+using MongoDB.Bson;
+using MongoDB.Driver.Core.Clusters;
+using MongoDB.Driver.Core.Misc;
+using MongoDB.Driver.Core.TestHelpers.XunitExtensions;
+using Xunit;
+
+namespace MongoDB.Driver.Examples.TransactionExamplesForDocs
+{
+    public class WithTransactionExample1
+    {
+        [Fact]
+        public void Example1()
+        {
+            RequireServer.Check().ClusterTypes(ClusterType.ReplicaSet, ClusterType.Sharded).Supports(Feature.Transactions);
+
+            var connectionString = CoreTestConfiguration.ConnectionString.ToString();
+            DropCollections(
+                connectionString, 
+                CollectionNamespace.FromFullName("mydb1.foo"), 
+                CollectionNamespace.FromFullName("mydb2.bar"));
+            var result = string.Empty;
```

Comment:
nit: We usually prefer `null` to indicate no value.

    string result = null;

Reply:
done